services/QuillLMS/app/models/schools_admins.rb

```diff
@@ -27,9 +27,8 @@ def admin
   end
 
   def send_admin_email
-    if Rails.env.production? || User.find(user_id).email.match('quill.org')
-      NewAdminEmailWorker.perform_async(user_id, school_id)
-    end
-  end
+    return unless Rails.env.production? || User.find(user_id).email.match('quill.org')
```

Comment:
This logic is a little hard to reason through as an `unless`, maybe: 
```suggestion
    staff_user = User.find(user_id)&.email&.match('quill.org')
    return unless (Rails.env.production? || staff_user)
```
(note, still don't love even my suggestion)